### DISCUSSION TITLE
Is there a version available to publish on Vercel?

### DISCUSSION THREAD

**Discussion by spencersmb** [colbyfayock/next-wordpress-starter#257] [Q&A] [answered]
I was really interested in this, but I can't seem to get ISG working on Netlify so I was wondering if there is a version that works on Vercel?

Thanks!
Spencer

**Comment by colbyfayock** [comment]
hey @spencersmb didn't realize this was failing on Vercel, but it should definitely be supported

in the meantime what issue are you having with Netlify? the project is deployed "as is" here: https://next-wordpress-starter-dev.netlify.app/

**Reply by spencersmb** [reply]
Hi @colbyfayock,
Thank you so much for responding, and creating this repo - I'm learning a ton! I was able to get the netlify version up and running just fine, however, when I wanted to switch for example the blog index page to ISR using the "revalidate" property and pushed the changes, I noticed the ISR didn't seem to work when I updated wordpress backend data. I thought that was all I needed to do when making a page go from Static Rendering it Incremental Static Rendering was to add the revalidate property to the return object of getStaticProps for a page. So that is what led me to try and publish it on the Vercel, but I ran into an error withe nextjs.config file if I remember correctly.

From what I have read, Netlify is working on a way to do ISR, but isn't ready just yet. That article was from earlier this year so maybe that has changed with a plugin possibly? Either way thank you again for any help and providing this repo. Side note (WpEngine released something called Atlas, which is really cool, but also costs an arm and leg :(

Cheers,
Spencer

**Reply by colbyfayock** [reply]
Thanks for letting us know, I fixed the issues where Netlify wouldn't be able to build the project with Next.js features. the only caveat is that you need to make sure the Essential Next.js Plugin is installed. That said, like you mentioned, I don't think ISR is currently working with the plugin, but other features should work now: https://github.com/colbyfayock/next-wordpress-starter/pull/262

As far as Vercel is concerned, that's still an issue, but I have a ticket open for that: https://github.com/colbyfayock/next-wordpress-starter/issues/259

if you have any additional info or want to try to debug / fix, feel free to comment on that Issue!

**Comment by spencersmb** [comment]
No problem! Glad to help in any way. In the meantime one issue I had going
from Netlify to Vercel is that the .env-local must have a prefix added to
it in order for the client side of Next.js to read the API URL when Apollo
is running. If all your ever doing is server-side queries then you don’t
need to worry, BUT I found that in order to get ISR working with Apollo,
for some reason it wouldn’t work unless the client side had access to the
Api URL variable. I’ll look at the PR you linked.

Cheers,
Spencer

**Reply by colbyfayock** [reply]
this should now also be fixed on vercel!

**Comment by colbyfayock** [comment, marked as answer]
Fixed via https://github.com/colbyfayock/next-wordpress-starter/pull/266
